platform/viewer/src/utils/OpenIdConnectRoutes.tsx

```diff
@@ -119,6 +128,15 @@ function OpenIdConnectRoutes({
 
   const navigate = useNavigate();
 
+  //for multi-tab logout
+  window.addEventListener('storage', () => {
+    var signOutEvent = localStorage.getItem("signoutEvent");
+    if (signOutEvent) {
+      navigate(`/logout?redirect_uri=${encodeURIComponent(window.location.href)}`);
+      localStorage.removeItem("signoutEvent");
+    }
+  });
+
```

Comment:
This even binding should happen in a useEffect on mount and the clean up be also in the return statement of it